devicetypes/smartthings/bose-soundtouch.src/bose-soundtouch.groovy

```diff
@@ -28,6 +28,8 @@ metadata {
         capability "Refresh"
         capability "Music Player"
         capability "Health Check"
+		capability "Sensor"
```

Comment:
Yes, Tabs :(
we eventually want to change everything to Tabs (my personal preference is spaces). But for now, I'll stick to consistency. This file has spaces, so the change should have spaces instead of tabs.